src/features/advanced_analytics_panel/components/AdvancedAnalyticsPanel/AdvancedAnalyticsPanel.module.css

```diff
@@ -63,3 +63,25 @@
   border-bottom: none;
   padding-bottom: 0;
 }
+
+@media screen and (max-width: 960px) {
+  .modalCover,
+  .modalCover * {
+    z-index: var(--modal);
+  }
+  .modalCover > section {
+    top: 44px; /* header offset */
+    left: 0;
+  }
+  .panel {
+    position: absolute;
+    margin: 0;
+    width: 100vw;
+    max-height: unset;
+    height: calc(100vh - 40px);
```

Comment:
we use this 40px everywhere, so maybe set it as css variable 🤔

Reply:
don't we set them somewhere in the ui kit? css var of header-height and mobile-header-height would be nice

Reply:
nope, we don't. 
I think it's okay to add it on the DN level, as needed only there.

Reply:
did that, very convinient, thanks!